src/components/Rocket/Rocket.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRocket, removeRocket } from '../../redux/rockets/rockets';
+import './Rocket.css';
+
+const Rocket = (props) => {
+  const {
+    id,
+    name,
+    image,
+    reserve,
+    description,
+  } = props;
+  const dispatch = useDispatch();
+  const reserveSpan = document.getElementById(id);
+
+  const handleClick = () => {
+    if (reserve) {
+      dispatch(removeRocket(id));
+      reserveSpan.classList.toggle('reserved');
+    } else {
+      dispatch(reserveRocket(id));
+      reserveSpan.classList.toggle('reserved');
+    }
```

Comment:
"Reserve Rocket" button updates the store successfully by dispatching an action 💪😎 Well done guys ✔